packages/flutter_tools/templates/module/ios/host_app_ephemeral_cocoapods/Podfile.copy.tmpl

```diff
@@ -8,13 +8,5 @@ target 'Runner' do
   install_flutter_plugin_pods flutter_application_path
 end
 
-post_install do |installer|
```

Comment:
This makes me so happy.  I believe we have some open issues around making this less painful for people right?  Or were those already resolved by moving it here instead of into the podhelper.rb?

Reply:
You're maybe thinking of issues like https://github.com/flutter/flutter/issues/24342.  I removed the post_install from the helper script with https://github.com/flutter/flutter/pull/36793/files#diff-b528f954e369c153bb31a631a8033be3L64, so I don't think it's painful anymore.  This one only runs for the ephemeral module, so no one should be editing it.

Reply:
It makes me happy too, though!